.travis.yml

```diff
@@ -12,6 +12,12 @@ cache:
     - $HOME/.composer/cache
     - repos/wordpress
 
+before_install:
+  - eval "$(ssh-agent -s)"
+  - echo -n "${base64_4950820e8b20_key}" | base64 --decode >.travis/key
+  - chmod 600 .travis/key
+  - ssh-add .travis/key
```

Comment:
If extracting the key to a secret (which seems to be what you're suggesting here), then the encrypted key in the repo is no longer needed either since `ssh-add` can read from STDIN (see [this answer](https://stackoverflow.com/a/46253163/1037938)).

Reply:
Done.